packages/flutter/lib/src/animation/curves.dart

```diff
@@ -6,11 +6,21 @@ import 'dart:math' as math;
 
 import 'package:flutter/foundation.dart';
 
-/// A mapping of the unit interval to the unit interval.
+/// An easing curve, i.e. a mapping of the unit interval to the unit interval.
```

Comment:
It explains it better than "easing curve" for people who understand math but not the term "easing curve"... I'm open to even better explanations though. :-)

Reply:
Maybe we can use the definition of easing in Material Design?

> Easing is a way to adjust an animation’s rate of change. Easing allows transitioning elements to speed up and slow down, rather than moving at a constant rate.
>
> In the real world, things don’t start or stop moving instantaneously. They take time to speed up and slow down. Easing makes elements move as though influenced by natural forces like friction."

https://material.io/design/motion/speed.html#easing


Alternatives I've seen from design website:

* "An easing curve is a line that defines the acceleration pattern on a graph." [1]
* "Easing functions specify the rate of change of a parameter over time." [2]



1. https://www.invisionapp.com/blog/crafting-easing-curves/
2. https://easings.net/